src/components/Article-votes.jsx

```diff
@@ -0,0 +1,34 @@
+import { React, useEffect, useState } from "react";
+import { incVote } from "../utils/api";
+
+const ArticleVotes = ({ votes, article_id }) => {
+  const [voted, setVoted] = useState(false);
+  const [votesState, setVoteState] = useState(0);
+  return (
+    <div>
+      <p>Votes: {votesState + votes} </p>
+      <button
+        disabled={voted}
+        onClick={() => {
+          incVote(1, article_id);
+          setVoteState(1);
+          setVoted(true);
+        }}
+      >
+        Up vote
+      </button>
+      <button
+        disabled={voted}
+        onClick={() => {
+          incVote(-1, article_id);
+          setVoteState(-1);
+          setVoted(true);
+        }}
+      >
+        Down Vote
+      </button>
```

Comment:
My only comment about the voting buttons is when I click either upvote or downvote, they both get disabled. What if a user has voted and wants to change their mind?